### PR TITLE
ci: publish AsyncAPI Studio Docker image with additional version tag

### PR DESCRIPTION
**Pull request by fgreinacher**
**Description**

This PR extends the Docker publishing workflow so that an additonal version tag is published to Docker Hub. This already happened before the migration from `semantic-release` to `changesets`. I think it's desirable to bring it back because there are only super outdated versions on Docker Hub right now: https://hub.docker.com/r/asyncapi/studio/tags.

**Related issue(s)**

Let me know if I should create one.

:hammer_and_pick: with :heart: by [Siemens](https://opensource.siemens.com/)

### CODE REVIEW

File: .github/workflows/publish-docker-image.yml [hunk 1]
```diff
@@ -17,10 +17,18 @@ jobs:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
 
+      - name: Get current Studio version
+        id: get-studio-version
+        uses: martinbeentjes/npm-get-version-action@v1.3.1
```
Comment (fgreinacher) [inline .github/workflows/publish-docker-image.yml:22]: See https://github.com/martinbeentjes/npm-get-version-action/tree/v1.3.1

File: .github/workflows/publish-docker-image.yml [hunk 2]
```diff
@@ -17,10 +17,18 @@ jobs:
...
         uses: docker/build-push-action@f2a1d5e99d037542a71f64918e516c093c6f3fc4
         with:
           context: .
           file: ./apps/studio/Dockerfile
           push: true
-          tags: asyncapi/studio:latest
\ No newline at end of file
+          tags: |
+            asyncapi/studio:latest
+            asyncapi/studio:${{ steps.get-studio-version.outputs.current-version}}
```
Comment (fgreinacher) [inline .github/workflows/publish-docker-image.yml:34]: Syntax as mentioned in https://github.com/docker/build-push-action?tab=readme-ov-file#inputs

**Comment by fgreinacher** [comment]
/ptal 

Let me know what I can do to make your review easier 🙇

File: .github/workflows/publish-docker-image.yml (hunk 1 above)
Comment (Shurtu-gal) [inline .github/workflows/publish-docker-image.yml:22]: Could you change the version to a commit SHA based version?
Something like `docker/build-push-action@f2a1d5e99d037542a71f64918e516c093c6f3fc4`

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

File: .github/workflows/publish-docker-image.yml (hunk 1 above)
Comment (fgreinacher) [reply .github/workflows/publish-docker-image.yml]: @Shurtu-gal Sure, done!

**Review by Shurtu-gal — APPROVED** [review]
LGTM

**Pull request merged**